department-of-veterans-affairs/va.gov-cms
# VACMS-8016 claro airtable fixes: buttons, spacing, breadcrumbs

## Description

Relates to #7746 & #8016. I honestly don't know where I got 7724 from for the branch number. Apologies for any confusion.

This PR:
- cleans up spacing between various elements on node view pages, the top portion should be consistently aligned regardless what subelements may be present (preview button/pending metadata/revisions/sidebar menus/breadcrumbs/etc.)
- allows tippy popups to render HTML tags (`<strong> <em>` etc.)
- fixes alignment of paragraphs buttons
- fixes dropbutton text & toggle alignment
- correct class on buttons on delete confirmation page
- correct class on buttons for adding various paragraphs
- updated border around paragraphs to be less attention grabbing, similar to vagovadmin (i changed it to too dark a gray earlier)
- moves breadcrumbs to first item below the blue header on every page type for consistency & familiarity to editors


## Screenshots
<img width="990" alt="Screen Shot 2022-02-14 at 9 01 47 AM" src="https://user-images.githubusercontent.com/11279744/153950010-0e141cfc-dd74-40d8-81fb-3d2cd63365ae.png">
<img width="1052" alt="Screen Shot 2022-02-14 at 9 01 52 AM" src="https://user-images.githubusercontent.com/11279744/153950018-c0940f84-8db5-43cc-8b0d-6fa2ffb4ab0a.png">
<img width="1033" alt="Screen Shot 2022-02-14 at 9 02 11 AM" src="https://user-images.githubusercontent.com/11279744/153950019-84d0d505-95eb-42f0-8418-7c9874e36bee.png">
<img width="1063" alt="Screen Shot 2022-02-14 at 9 02 20 AM" src="https://user-images.githubusercontent.com/11279744/153950024-483c7b54-cfde-46de-9dc2-d89112a06510.png">
<img width="733" alt="Screen Shot 2022-02-14 at 9 33 22 AM" src="https://user-images.githubusercontent.com/11279744/153950026-96b679f9-7701-4d98-90e6-c6e359a6ab91.png">
<img width="312" alt="Screen Shot 2022-02-14 at 10 57 53 AM" src="https://user-images.githubusercontent.com/11279744/153950027-7df17251-ad36-453a-ab3c-2226c11c989a.png">
<img width="260" alt="Screen Shot 2022-02-14 at 11 29 11 AM" src="https://user-images.githubusercontent.com/11279744/153950028-6870268c-7008-4669-a56b-2a8fb2770b5b.png">
<img width="751" alt="Screen Shot 2022-02-14 at 11 48 06 AM" src="https://user-images.githubusercontent.com/11279744/153950032-0a210857-479e-45cc-8933-d51dd1fbe165.png">
<img width="1026" alt="Screen Shot 2022-02-14 at 12 43 46 PM" src="https://user-images.githubusercontent.com/11279744/153950034-c329942a-340a-4a23-bdf5-139a44dee214.png">
<img width="511" alt="Screen Shot 2022-02-14 at 12 44 32 PM" src="https://user-images.githubusercontent.com/11279744/153950035-c4805da9-c4df-42b3-a751-f731d9077d87.png">



## QA steps
- log in as `witt.cook@va.gov`
- see that breadcrumbs are now in the white area of the page for vet centers
- log out, log in as admin
- visit `node/2363/edit`
- see that all paragraphs buttons are small size
- visit `node/add/health_care_region_detail_page`
- see that the button spacing for the 3 different paragraphs fields is consistent
- add a q&a paragraph to featured block
- see that all triple dot paragraph action toggles are aligned
- visit `health-care/health-needs-conditions`
- see that tippy tooltips render html tags correctly, no tags print in the plaintext
- see that the border around paragraphs is lighter and closer to what vagovadmin is doing
- visit `admin/content-models/content`
- see that the dropbuttons text and toggle button is aligned

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
